settings.gradle

```diff
@@ -53,6 +53,7 @@ includeWithFlags ':junit5',                              'java', 'publish', 'rel
 includeWithFlags ':kafka',                               'java', 'publish', 'relocate'
 includeWithFlags ':kotlin',                              'java', 'publish', 'relocate', 'kotlin'
 includeWithFlags ':logback',                             'java', 'publish', 'relocate'
+includeWithFlags ':logback13',                           'java', 'publish', 'relocate', 'no_aggregation'
```

Comment:
added `no_aggregation` to avoid `:javadoc:javadoc` errors.
Please let me know if I misunderstood

Reply:
no_aggregation is a correct choice here.